Services/AuthenticationService/Controllers/AuthenticationController.cs

```diff
@@ -31,7 +33,9 @@ public string Login([FromServices] ILoginCommand command, [FromBody] UserEmailPa
         {
             var result = command.Execute(user);
 
-            return result.Result;
+            var userToken = result.Result;
+
+            return JsonSerializer.Serialize(userToken);

```

Comment:
Why don't we check id here?  If we check it in dbs it is solid error. Why don't we get useremailpassword by email from database and then check if password hash the same here

Reply:
> Why don't we check id here? If we check it in dbs it is solid error. Why don't we get useremailpassword by email from database and then check if password hash the same here

Why would we need to check id anywhere?
Client sends email and password hash and gets the id and token, which are correct because they come from DBS